redbot/cogs/general/general.py

```diff
@@ -216,17 +216,19 @@ def __init__(self):
         created_on = _("{}\n({} days ago)").format(user_created, since_created)
         joined_on = _("{}\n({} days ago)").format(user_joined, since_joined)
 
-        game = _("Chilling in {} status").format(user.status)
-
-        # Check if the user has a special status
-        if user.game.type == 0:  # "Playing" status
+        
+        if user.game is None:  # Default status
+            game = _("Chilling in {} status").format(user.status)
+        elif user.game.type == 0:  # "Playing" status
             game = _("Playing {}").format(user.game.name)
         elif user.game.type == 1:  # "Streaming" status
             game = _("Streaming [{}]({})").format(user.game.name, user.game.url)
         elif user.game.type == 2:  # "Listening" status
             game = _("Listening to {}").format(user.game.name)
         elif user.game.type == 3:  # "Watching" status
             game = _("Watching {}").format(user.game.name)
+        else:
+            game = "please go bother <@111655405708455936> as somehow something broke"
```

Comment:
I don't like this, I'd prefer the command threw an error so the user can submit the traceback. Please remove the `else` block

Reply:
Wait a second, don't remove the `else` block. I'd rather have it throw an error (like a `ValueError` or `RuntimeError`) or just display the default "chilling" message as Will's example provides.